namespaces/aushape.yml

```diff
@@ -0,0 +1,55 @@
+
+namespace:
+  name: aushape
+  type: group
+  description: >
+    Audit events converted with aushape
+    For more information please see
+    https://github.com/Scribery/aushape
+  fields:
+  - name: serial
+    type: long
+    description: >
+      Audit event serial number
+  - name: time
+    type: date
+    description: >
+      Audit event timestamp
+  - name: node
+    type: string
+    description: >
+      Name of the host where the audit event occurred
```

Comment:
This can be different, if the event was delivered to the originating host using audisp-remote and then converted to JSON here and logged via ViaQ collector. It might be useful to keep it separate.